EBH-4581/query_spliceai_and_gtex.py

```
@@ -0,0 +1,337 @@
+            DataFrame containing SpliceAI scores per variant, including:
+            - Variant
+            - Input_HGVSc
+            - SpliceAI_transcript_set
+            - SpliceAI_DS_AL, SpliceAI_DP_AL
+            - SpliceAI_DS_DL, SpliceAI_DP_DL
+            - SpliceAI_DS_AG, SpliceAI_DP_AG
+            - SpliceAI_DS_DG, SpliceAI_DP_DG
+    """
+    results = []
+
+    for idx, row in excel_file.iterrows():
```

Ruff (0.15.2)

[warning] Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)